sidebars.js

```diff
@@ -12,7 +12,15 @@ module.exports = {
         'create/index-create',
         'create/rediscloud/index-rediscloud',
         'create/heroku/index-heroku',
-        'create/azure/index-azure',
+          {
+              type: 'category',
+              label: 'Redis on Azure Cache',
+              items: [
+                  'create/azure/index-azure',
+                  'create/azure/portal/index-azure-portal',
```

Comment:
missing reference to /create/azure/terraform-private-endpoint

Reply:
Let's get it added once tutorial is completed.